proto/streamlit/proto/Skeleton.proto

```diff
@@ -19,6 +19,17 @@ syntax = "proto3";
 option java_package = "com.snowflake.apps.streamlit";
 option java_outer_classname = "SkeletonProto";
 
-// An internal-only element that displays an app skeleton.
+enum SkeletonStyle {
+    PLAIN = 0;
+    APP = 1; // internal-only, for now
+}
+
+// An empty-like element that displays an app skeleton.
 message Skeleton {
+    // Height in CSS points
+    optional int32 height = 1;
+    // Width as a fraction 0 - 1
+    optional double width = 2;
+    // Skeleton visual style
+    optional SkeletonStyle style = 3;
```

Comment:
I don't think this should be `optional`. I think we should always provide the desired style.